schemes/rrtmgp/rrtmgp_pre.meta

```diff
@@ -0,0 +1,301 @@
+  dimensions = ()
+  intent = in
+[ offset ]
+  standard_name = number_of_seconds_until_next_shortwave_radiation_timestep
+  units = s
+  type = integer
+  dimensions = ()
+  intent = out
+[ idxday ]
+  standard_name = daytime_points
+  units = index
+  type = integer
+  dimensions = (horizontal_dimension)
+  intent = out
+[ nday ]
+  standard_name = daytime_points_dimension
+  units = count
+  type = integer
+  dimensions = ()
+  intent = out
+[ idxnite ]
+  standard_name = nighttime_points
+  units = index
+  type = integer
+  dimensions =  (horizontal_dimension)
+  intent = out
+[ nnite ]
+  standard_name = nighttime_points_dimension
+  units = count
+  type = integer
+  dimensions = ()
+  intent = out
+  dimensions = ()
+[ errmsg ]
+  standard_name = ccpp_error_message
+  long_name = Error message for error handling in CCPP
+  units = none
+  type = character | kind = len=512
+  dimensions = ()
+  intent = out
+[ errflg ]
+  standard_name = ccpp_error_code
+  long_name = Error flag for error handling in CCPP
+  units = 1
+  type = integer
+  dimensions = ()
+  intent = out
+
+
+[ccpp-arg-table]
+  name  = rrtmgp_pre_run
+  type  = scheme
+[ coszrs ]
+  standard_name = cosine_of_solar_zenith_angle_for_radiation
+  units = 1
+  type = real | kind = kind_phys
+  dimensions = (horizontal_loop_extent)
+  intent = in
+[ nstep ]
+  standard_name = current_timestep_number
+  units = count
+  type = integer
+  dimensions = ()
+  intent = in
+[ dtime ]
+  standard_name = timestep_for_physics
+  units = s
+  type = real | kind = kind_phys
+  dimensions = ()
+  intent = in
+[ iradsw ]
+  standard_name = frequency_of_shortwave_radiation_calculation
+  units = 1
+  type = integer
+  dimensions = ()
+  intent = in
+[ iradlw ]
+  standard_name = frequency_of_longwave_radiation_calculation
+  units = 1
+  type = integer
+  dimensions = ()
+  intent = in
+[ irad_always ]
+  standard_name = number_of_timesteps_to_force_radiation_calculation_after_initialization
+  units = count
+  type = integer
+  dimensions = ()
+  intent = in
+[ ncol ]
+  standard_name = horizontal_loop_extent
+  units = count
+  type = integer
+  dimensions = ()
+  intent = in
+[ next_cday ]
+  standard_name = fractional_calendar_days_on_end_of_next_timestep
+  units = 1
+  type = real | kind = kind_phys
+  dimensions = ()
+  intent = in
+[ idxday ]
+  standard_name = daytime_points
+  units = index
+  type = integer
+  dimensions = (horizontal_loop_extent)
+  intent = in
+[ nday ]
+  standard_name = daytime_points_dimension
+  units = count
+  type = integer
+  dimensions = ()
+  intent = in
+[ idxnite ]
+  standard_name = nighttime_points
+  units = index
+  type = integer
+  dimensions =  (horizontal_loop_extent)
+  intent = in
+[ nnite ]
+  standard_name = nighttime_points_dimension
+  units = count
+  type = integer
+  dimensions = ()
+  intent = in
+[ dosw ]
+  standard_name = do_shortwave_radiation
+  units = flag
+  type = logical
+  dimensions = ()
+  intent = out
```

Comment:
Can we move these `intent(out)` variables so that they come after all of the `intent(in)` variables?